.env.template

```diff
@@ -2,33 +2,35 @@ PRIVATE_KEY=0xabc123abc123abc123abc123abc123abc123abc123abc123abc123abc123abc1
 
 # Network configuration
 RPC_URL_L1=http://65.108.230.142:8545
-RPC_URL_L2=https://rpc.beta.testnet.l2.quarkchain.io:8545
-BLOCKSCOUT_API_URL=https://explorer.beta.testnet.l2.quarkchain.io/api
 ETHERSCAN_API_KEY=ABC123ABC123ABC123ABC123ABC123ABC1
 
 # Contract configuration
 MAX_KV_SIZE_BITS=17 # maxKvSizeBits, 131072
 SHARD_SIZE_BITS=39 # shardSizeBits ~ 512G
 RANDOM_CHECKS=2 # randomChecks
-CUTOFF=7200 # cutoff = 2/3 * target internal (3 hours), 3 * 3600 * 2/3
+CUTOFF=28800 # cutoff = 2/3 * target internal (12 hours), 12 * 3600 * 2/3
 DIFF_ADJ_DIVISOR=32 # diffAdjDivisor
-TREASURY_SHARE=100 # treasuryShare, means 1%
+TREASURY_SHARE=2500 # treasuryShare, means 25%
 
 # Constructor parameters
-STORAGE_COST_L1=1500000000000000 # storage cost per blob - https://ethresear.ch/t/ethstorage-scaling-ethereum-storage-via-l2-and-da/14223/6#incentivization-for-storing-m-physical-replicas-1
-STORAGE_COST_L2=570000000000000000
+STORAGE_COST_L1=60000000000000 # storage cost per blob 60000Gwei
 DCF_FACTOR=340282366367469178095360967382638002176 # dcfFactor, 0.95 for yearly discount
-UPDATE_LIMIT=90 # 45 blobs/s according to sync/encoding test, times block interval of L2
 
 # Initialize parameters
-MINIMUM_DIFF_L1=4718592000 # 5 * 3 * 3600 * 1024 * 1024 / 12 = 4718592000 for 5 replicas that can have 1M IOs in one epoch
-MINIMUM_DIFF_L2=94371840 # 0.1 * 3 * 3600 * 1024 * 1024 / 12 = 94371840 for 0.1 replicas that can have 1M IOs in one epoch
-PREPAID_AMOUNT_L1=3145728000000000000000 # prepaidAmount - 50% * 2^39 / 131072 * 1500000 Gwei, it also means 3145 ETH for half of the shard
-PREPAID_AMOUNT_L2=119537664000000000000000 # prepaidAmount - 50% * 2^39 / 131072 * 570000000 Gwei, it also means around 1,200,000 QKC for half of the shard
+MINIMUM_DIFF_L1=377487360 # 0.1 * 12 * 3600 * 1024 * 1024 / 12 for 0.1 replicas that can have 1M IOs in one epoch
```

Comment:
What is the reason behind the choice of 0.1 replica?

Reply:
MINIMUM_DIFF_L1 also serves as the initial difficulty at launch. Choosing a relatively small value allows the first block to be mined more quickly, which helps verify that the launch was successful.